pkg/kp/pcstore/consul_store.go

```diff
@@ -151,3 +162,72 @@ func kvpToPC(pair *api.KVPair) (fields.PodCluster, error) {
 
 	return pc, nil
 }
+
+// Watch implements a watch for the Pod cluster at _id_
+// It will return a blocking channel on which the client can read
+// WatchedPodCluster objects. The goroutine maintaining the watch will block on
+// writing to this channel so it's up to the caller to read it with haste.
+// One blocking watch will not affect any of the other watches.
+func (s *consulStore) Watch(id fields.ID, quit <-chan struct{}) <-chan WatchedPodCluster {
```

Comment:
We'll want to use it to filter the returned pod clusters, so the answer to your question is "no, not yet".